### PR TITLE
Goa 3241 rationalise proteome related properties

### PR DESCRIPTION
**Pull request by rizwan-ishtiaq**

### CODE REVIEW

**Comment by rizwan-ishtiaq** [comment]
Suggested changes has made and commited

File: geneproduct-rest/src/main/java/uk/ac/ebi/quickgo/geneproduct/model/GeneProductRequest.java [hunk 1]
```diff
@@ -66,25 +66,22 @@
...
-            " product is not assigned to a proteome, but is a protein. A proteomeMembership of 'Not applicable'" +
-            " means the gene product is not a protein, and cannot be part of a proteome.",
-            allowableValues =
-                    "Reference," +
-                            "Complete,None,Not applicable",
-            example =
-            "Reference")
-    private String proteomeMembership;
+    @ApiModelProperty(value = "Filters the results of the main query based on a value chosen from the proteome field." +
+            " Proteins with a proteome 'gcrpCan' (aka reference) are part of a subset of proteomes that have been " +
```
[the remark this answers is not in the document]
Comment (TonyWardell) [reply geneproduct-rest/src/main/java/uk/ac/ebi/quickgo/geneproduct/model/GeneProductRequest.java]: remove aka reference or

**Comment by TonyWardell** [comment]
Just get rid of the aka reference as above and then merge to master

**Comment by rizwan-ishtiaq** [comment]
Now This is ready to merge into master

**Pull request merged**